### ISSUE TITLE
Using powerpoint presentation with Swedish layout names

### ISSUE THREAD

**Issue by mikaelmoutakis** [davidgohel/ReporteRs#57]
```
> pptx(template="test_bug_presentation.pptx")
Error in pptx(template = "test_bugg_presentation.pptx") : 
  Following layout names are invalid, they should only contain spaces, letters (from a to z) and numeric values :
'ÖÅÖ layout', 'Lodrät rubrik och text', 'öljsdföjsdöfj', 'Innehåll med bildtext', 'Två innehållsdelar', 'Jämförelse', 'Rubrik och lodrät text', 'Rubrik och innehåll'
```
Powerpoint see these as valid layout names. I could send you a powerpoint presentation, but I don't know how. 

I'm using 
R version 3.1.1 (2014-07-10) -- "Sock it to Me"
Copyright (C) 2014 The R Foundation for Statistical Computing
Platform: x86_64-apple-darwin13.1.0 (64-bit)

and
ReporteRs from GitHub (downloaded yesterday)

**Comment by mikaelmoutakis** [comment]
Thank you, but having to reformat all Company PP templates is a pain. And åäö etc are real letters.

**Comment by davidgohel** [comment]
Hi

I just solved that issue (version 0.8.0)

David

**Comment by mikaelmoutakis** [comment]
Thanks a lot! I really appreciate it. I'll file a bug report if I encounter any problems.
